packages/backend/src/managers/playgroundV2Manager.ts

```diff
@@ -81,40 +82,32 @@ export class PlaygroundV2Manager extends Publisher<PlaygroundV2[]> implements Di
     return `playground-${++this.#UIDcounter}`;
   }
 
-  createConversation(): string {
-    return this.#conversationRegistry.createConversation();
-  }
-
   /**
-   * @param containerId must be corresponding to an inference server container
-   * @param modelId the model to use, should be included in the inference server matching the containerId
-   * @param conversationId the conversation id to happen the message to.
+   * @param playgroundId
    * @param userInput the user input
    * @param options the model configuration
    */
-  async submit(
-    containerId: string,
-    modelId: string,
-    conversationId: string,
-    userInput: string,
-    options?: ModelOptions,
-  ): Promise<void> {
-    const server = this.inferenceManager.get(containerId);
+  async submit(playgroundId: string, userInput: string, options?: ModelOptions): Promise<void> {
```

Comment:
submit should be changed to a sync method as we should not be waiting on its result. For example

```ts

submit(playgroundId: string, userInput: string, options?: ModelOptions): void {
// in a sync maner ensure conversation exist

// do not wait for the result
process(playgroundId, userInput, options).catch(err => ...)
}

private process(playgroundId: string, userInput: string, options?: ModelOptions): Promise<void> {
// do usual logic
}
```

Reply:
Otherwise we get the timeout error while we are getting the conversations responses
<img width="559" alt="image" src="https://github.com/projectatomic/ai-studio/assets/42176370/64a645bb-8302-43aa-a90a-df98cf186dcb">